src/components/StringField.vue

```diff
@@ -119,6 +118,11 @@ export default {
             })
             .catch(_ => _)
         },
+        fieldHasKeyword: function(keyword) {
```

Comment:
nitpick, but why are you using different syntaxes for function definitions in methods and computed? Cf https://github.com/etalab/csv-gg/blob/7bf9484ca64d78ce2fd98c592f266c6495e69ac0/src/components/StringField.vue#L137

Reply:
I tend to prefer the shorter version but it's not a requirement.

Reply:
Changed in the next commit 👍